README.md

```diff
@@ -29,3 +29,16 @@ This repo contains some useful bash scripts which are making my day-to-day life
 		usage example: bash ping_network.sh
 		after execute the script pass you network
 		eg: 192.168.0.1
+
+## 6.udit
+        A simple script that takes contest name as input and goes to that folder, removes previous files, creates new files using predefined template.
+        
+	Usage
+	
+        copy file udit to folder "/usr/local/bin/" and file1.cpp, file2.cpp to "~/Documents/Format/". Run command udit <Contest_Name>. Add these lines to .bashrc
```

Comment:
Why to add /usr/local/bin you can add the directory in PATH variable

Reply:
Instead of moving file file1.cpp file2.cpp you can use cvim template

Reply:
We'll, there are multiple ways, you gotta do it some way. IMO my way is easy to understand and easy to implement.